tests/python/unittest/test_numpy_op.py

```diff
@@ -597,6 +597,98 @@ def _test_np_exception(func, shape, dim):
             else:
                 _test_np_exception(func, shape, dim)
 
+@with_seed()
```

Comment:
One more blank line above.
2-line gaps between Python functions is recommended.